zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json

```diff
@@ -1099,5 +1124,172 @@
         ]
       }
     ]
+  },
+  {
+    "type": "fieldset",
+    "title": "Dataset",
+    "description": "Uhm.",
+    "fieldhint": "optional",
+    "collapsed": true,
+    "items": [
+      {
+        "type": "array",
+        "key": "locations",
+        "title": "Locations",
+        "fa_cls": "fa-tags",
```

Comment:
Let's find a decent icon for these guys, e.g. `fa-map-marker` for the locations, maybe a calendar for the dates, etc.